templates/base.html

```diff
@@ -21,6 +21,7 @@
     integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">
   <script src="https://kit.fontawesome.com/6803f19786.js" crossorigin="anonymous"></script>
   <link rel="stylesheet" href="{% static 'css/homePage.css' %}">
```

Comment:
Suggestion: The index-specific stylesheet could be added within the `happiness_app`app's static folder so that iti is scoped to there.